src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheckTest.java

```diff
@@ -473,7 +473,7 @@ public void testLeftCurlyRecordsAndCompactCtors() throws Exception {
             "52:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
             };
         verify(checkConfig,
-            getNonCompilablePath("InputLeftCurlyRecordsAndCompactCtors.java"), expected);
+            getNonCompilablePath("InputLeftCurlyTestRecordsAndCompactCtors.java"), expected);
```

Comment:
please correct config comment in the corresponding input file

Reply:
Done👍